states/massachusetts/elections.toml

```diff
@@ -8,7 +8,87 @@ date = 2020-09-01
 name = "Congressional Primary"
 original_date = 2020-09-01
 
+[20200901.registration]
+in_person_by = 2020-08-22
+online_by = 2020-08-22T23:59:00
+postmarked_by = 2020-08-22
+sources = [
+  "http://www.sec.state.ma.us/ele/eleifv/howreg.htm",
+  "https://www.sec.state.ma.us/ele/elepdf/2020-Primary-and-Election-Schedule-Legal-Sized.pdf",
+]
+# Note the Day to Register was extended to 10 days from 20 days prior in 2020
+
+[20200901.absentee.application]
+email_by = 2020-10-31T12:00:00
+fax_by = 2020-10-31T12:00:00
+received_by = 2020-10-31T12:00:00
+# Because you can vote absentee in-person day before is OK
+sources = ["http://www.sec.state.ma.us/ele/eleabsentee/absidx.htm"]
+
+[20200901.absentee]
+in_person_by = 2020-09-01 # You must make an appointment
+received_by = 2020-09-01 
+sources = ["http://www.sec.state.ma.us/ele/eleabsentee/absidx.htm"] 
+
+[20200901.poll.early.application]
+# Mail in Early Voting Application
```

Comment:
I think I'd have this be .absentee because it is broadly available and matches the messaging we already have. Instead I'd make "absentee" be "emergency" since its geared towards last minute, unexpected events.

Reply:
Hmm... so we'll be sending out reminders for "Emergency" stuff but in MA, they call it absentee.

I don't know the right answer, but we've conflated mail-in with absentee and we've assumed all states use terminology the same; they have the same process and deadlines.
We could have more deadline types and language described globally and only used only where needed..
We could move those definitions to the state info.toml instead, let them locally defined.

None of which sounds like fun to change.

I'm going to make the change you describe. We can refactor later when we learn the other cases we don't know about yet. Like we just learned this one. ;-)

Reply:
OK... changes on the way... waiting for CI